automated-deployments-using-github-actions.md

```diff
@@ -0,0 +1,300 @@
+# Automating Okteto cloud workflows using Okteto Actions and GitHub Actions.
+
+## Introduction
+
+GitHub Actions provides developers with the functionality of building and automating  the execution of their software development workflows directly in a project’s repository.
+
+In this tutorial you will learn how you can automate the workflow of an existing Flask API whose continuous integration process is managed using [GitHub Actions](https://github.com/features/actions). You will learn about the Okteto actions and how you can use them to work with Okteto from a GitHub Action build.
+
+
+## Why Automate Your Workflow?
+
+The [Okteto CLI](https://okteto.com/docs/getting-started/installation/index.html) does a fantastic job at simplifying the process of working with your Okteto resources. However, over time, executing commands to build and deploy your docker image to the Okteto cloud after major changes could become repetitive and tiring. 
+
+According to [Google’s SRE](https://sre.google/) practices, a way to [eliminate the toil time](https://sre.google/sre-book/eliminating-toil/) resulting from such operations is to automate the execution of such commands through the use of scripts or an external automation service. 
+
+For this article, we will be using [GitHub Actions](https://github.com/features/actions) as an automation tool to build a docker image from merged code changes and deploy to the docker image to the Okteto Cloud using the available [Okteto Actions](https://github.com/okteto/actions).
+
+## Prerequisites 
+
+In order to follow along with this tutorial, it is expected that you satisfy the following requirements;
+
+
+- Have an [Okteto Cloud](https://okteto.com/) Account
+- Have the [Okteto CLI](https://okteto.com/docs/getting-started/installation/index.html) installed on a host machine
+- Have a [GitHub account](https://github.com/) with Git installed on a host machine
+- Have a basic understanding of the Python programming language
+- Have an installation of [Docker](https://www.docker.com/) on your local machine.
+
+
+## Step 1: Clone a Sample Python Flask App
+
+To get started, fork the sample application from it’s repository [here](https://github.com/vickywane/okteto-flask-app). After forking the repository, execute the command below from your terminal to clone your forked copy of the repository to your host machine;
+
+**Note**: *Replace the* `*GITHUB_USERNAME*` *in the URL below with your GitHub username to match the remote origin of the forked repository.* 
+
+
+    git clone https://github.com/{{GITHUB_USERNAME}}/okteto-flask-app
+
+ 
+Going through the `flaskr.py` file within the cloned `okteto-flask-app` folder in your preferred code editor, you would observe that it contains three API routes that are used for performing a create, retrieve and delete operation against a connected Couch database.
```

Comment:
"In the flaskr.py, there are three routes defined...".

Reply:
Alright. I have rephrased the sentence into this.